Applications/InciSe/CMakeLists.txt

```diff
@@ -8,44 +8,26 @@ endif()
 # Create a cache entry for the provisioning file which is used to export
 # the file name in the MITKConfig.cmake file. This will keep external projects
 # which rely on this file happy.
-set(MITK_InciSe_PROVISIONING_FILE "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/MitkInciSe.provisioning" CACHE INTERNAL "InciSe provisioning file" FORCE)
+set(MITK_InciSe_PROVISIONING_FILE "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/InciSe.provisioning" CACHE INTERNAL "InciSe provisioning file" FORCE)
 
 set(_plugins
  org.mitk.gui.qt.stdmultiwidgeteditor
  org.mitk.gui.qt.datamanager
  org_mitk_planarfigure
+ org.blueberry.ui.qt.help
 
  upenn.cbica.captk.ui
  upenn.cbica.captk.application
+ upenn.cbica.captk.common
  upenn.cbica.captk.interactivesegmentation
+
  org.mitk.gui.qt.imagenavigator
  org_mitk_gui_qt_imagenavigator
```

Comment:
what is the difference between this and the previous line.

Reply:
I got this part from MITK. I'm afraid to delete one of them because it might break MAC or something.